packages/terra-dropdown-button/src/_DropdownList.module.scss

```
@@ -0,0 +1,14 @@
+:local {
+  .dropdown-list {
+    background: var(--terra-dropdown-button-menu-background, #fff);
```

Let's update the variable names like so
```suggestion
    background: var(--terra-dropdown-list-background, #fff);
```

---

Stylelint gets upset when the `button-` is removed because then the variable doesn't begin with `--terra-dropdown-button`